src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs

```diff
@@ -327,6 +327,7 @@ public async Task CloseAsync_DuringConcurrentReceiveAsync_ExpectedStates(Uri ser
 
         [ConditionalFact(nameof(WebSocketsSupported))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Browser)]
```

Comment:
These can all be 

```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows | TestPlaforms.Browser, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Browser)]
```

But I also made an error in my first review comment and meant https://github.com/dotnet/runtime/issues/42852

Reply:
Addressed in recent commits